composer/trainer/trainer.py

```diff
@@ -491,6 +491,25 @@ class Trainer:
             Ignored if ``load_path`` is either ``None`` or a local file path. (default: ``1,048,675``)
         load_progress_bar (bool, optional): Display the progress bar for downloading the checkpoint.
             Ignored if ``load_path`` is either ``None`` or a local file path. (default: ``True``)
+        load_ignore_keys (List[List[str]] | (Dict) -> None, optional): A list of paths for the ``state_dict``,
+            which, when provided, will be ignored from the state_dict before a checkpoint is loaded. Each path is a list
+            of strings specifying the keys to index into ``state_dict``. If a prefix is provided, all children are also
+            ignored.
+
+            Example 1: `load_ignore_model_keys = [["state", "model", "classifier", "weights"], "state", "model", "classifier", "bias"]]`
```

Comment:
YAHP won't be able to parse a nested list, so using a string here with a seperator would be preferred. However, thoughts about using `"/" to denote nested sub-dictionaries instead of "." to be more consistent with glob syntax (and I think pytorch natively uses "." in model parameter names?) Then, each path would be a glob string.

Reply:
I recall in the discussion that `.` feels more natural to our users to align with how pytorch indexes into nested `nn.Module`. "/" might be confused with filepath directories.